public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -29,9 +31,6 @@ include ../../../../_includes/_util-fns
   * [Template expression operators](#expression-operators)
 // #enddocregion intro
 
-:marked
-  [Run the live example](/resources/live-examples/template-syntax/ts/plnkr.html)
```

Comment:
Disagree about removing the example which is enormously valuable in exploring the ideas expressed here and closely tracks the ToC here.

But I do agree that the purpose should be made clear. Doing that.